smarttender.robot

```
@@ -625,7 +631,7 @@ Input Ade
     sleep  2s
     Input text  jquery=div#lotAmount0 input  ${value}
     Click Element  jquery=button#submitBidPlease
-    Wait Until Page Contains  Пропозицію прийнято  15s
+    run keyword and ignore error  Wait Until Page Contains  Пропозицію прийнято  60s
```

Run Keyword And Ignore Error